.changeset/dirty-bulldogs-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/website': patch
```

Comment:
We don't really commit changesets for website changes. This and theme-designer aren't published to npm, so we don't give them changesets. It'll solve the dangerjs issue too